File: src/CSharp/601-700/674.LongestContinuousIncreasingSubsequence.cs

using System;
using System.Linq;
using System.Collections.Generic;

namespace PeefyLeetCode.FindLengthOfLCIS
{
public class Solution {
    public int FindLengthOfLCIS(int[] nums) {
        int count = 0;
        int maxcount = 0;
        int last = -9999999;
        foreach (var num in nums){
            if (num > last)
                count += 1;
            else
                count = 1;
            if (count > maxcount)
                maxcount = count;
            last = num;
        }
        return maxcount; 
    }
}
}

File: src/Python/601-700/674.LongestContinuousIncreasingSubsequence.py

class Solution:
    def findLengthOfLCIS(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        maxcount = 0
        last = -9999999
        for num in nums:
            if num > last:
                count += 1
            else:
                count = 1
            maxcount = max(maxcount, count)
            last = num
        return maxcount 
                
if __name__ == '__main__':
    solution = Solution()
    print(solution.findLengthOfLCIS([1, 3, 5, 4, 7]))
else:
    pass
